index.html

```diff
@@ -1,9 +1,108 @@
-NPM Clone Nifty Penguin Magic npm Enterprise Products Solutions Resources Docs Support Search Join Log In Build amazing
-things Essential JavaScript development tools that help you go to market faster and build powerful applications using
-modern open source code. See plans Join for free Bring the best of open source to your company npm is the tool used by
-over 11,000,000 JavaScript developers around the world. Your developers already use it. Your company depends on it.
-Create an Org and get more out of the tools your team already knows and loves. Zero configuration Create an org, add
-your team, and start collaborating. Nothing to configure, set up, or manage. Team management Control who has access to
-what modules within your team namespace using straightforward team management capabilities. Familiar features npm Orgs
-has 100% parity with all the public npm registry features your developers already use. npm audit Enjoy the security
-auditing features built into the npm client, a zero-friction way to make open source software safer. Create an Org
+<head>
+    <link rel="stylesheet" href="./style.css" />
```

Comment:
Remember to put between the < head> < /head> brackets:

< meta charset="UTF-8"/ >
<title>npm | build amazing things</title>


They are important. Meta serves to read the code and symbols in the character encoding we choose (usually we'll be using UTF-8). Title is useful to put a name to the tab of your browser. 

<img width="260" alt="Untitled 2" src="https://user-images.githubusercontent.com/66212802/95980227-20ad4980-0e1d-11eb-9c53-c61bc6c9b178.png">